playerbot/strategy/triggers/GenericTriggers.h

```diff
@@ -418,6 +425,28 @@ namespace ai
         Trigger* rs;
     };
 
+    class OrTrigger : public Trigger
+    {
+    public:
+        OrTrigger(PlayerbotAI* ai, Trigger* ls, Trigger* rs) : Trigger(ai)
+        {
+            this->ls = ls;
+            this->rs = rs;
+        }
+        virtual ~OrTrigger()
+        {
+            delete ls;
+            delete rs;
+        }
+    public:
+        virtual bool IsActive();
+        virtual string getName();
+
+    protected:
+        Trigger* ls;
+        Trigger* rs;
+    };
+

```

Comment:
Is it still used? Or is it just for rounding off possible triggers?

Reply:
I was used at the beginning, but I did not remove it, of the "keep for rounding" purposes